include/CLI/impl/App_inl.hpp

```diff
@@ -1496,10 +1514,20 @@ CLI11_INLINE void App::_parse(std::vector<std::string> &args) {
         // Convert missing (pairs) to extras (string only) ready for processing in another app
         args = remaining_for_passthrough(false);
     } else if(parse_complete_callback_) {
+        _process_callbacks(CallbackPriority::FirstPreHelp);
+        _process_help_flags(CallbackPriority::First);
```

Comment:
A last comment: shouldn't we also call
```suggestion
        _process_help_flags(CallbackPriority::FirstPreHelp);
```
for all help flag processing? In case a user sets the priority of the help flags to `CallbackPriority::*PreHelp`, the help callback wouldn't be called otherwise.